.github/workflows/release.yml

```diff
@@ -29,13 +29,17 @@ jobs:
         GOOS: ${{ matrix.os }}
         GOARCH: ${{ matrix.arch }}
       run: make release-${{ matrix.target }}
+    - name: Making helm charts
+      run: make release-chart
     - name: Uploading assets...
       if: ${{ !env.ACT }}
       uses: softprops/action-gh-release@v1
       with:
         files: |
           _output/release/${{ matrix.target }}-${{ matrix.os }}-${{ matrix.arch }}.tgz
           _output/release/${{ matrix.target }}-${{ matrix.os }}-${{ matrix.arch }}.tgz.sha256
+          _output/release/karmada-chart-*.tgz
+          _output/release/karmada-chart-*.tgz.sha256
```

Comment:
Please check if the output msg is correct.
`karmada-v1.5.0.tgz` vs `karmada-chart-v1.5.0.tgz`.
```
# git rebase master
First, rewinding head to replay your work on top of it...
Applying: upload helm charts in CI pipeline
-bash-5.0# make release-chart
hack/release-helm-chart.sh 'v1.5.0-31-g22a352be'
Preparing: 'helm' existence check - passed
Starting to package into a Karmada chart archive



Successfully packaged chart and saved it to: _output/release/karmada-v1.5.0.tgz

```